tests/test_config_checker.py

```
@@ -281,6 +281,300 @@ def test_check_config_valid_ble(
+
+    @patch("mmrelay.cli.parse_arguments")
+    @patch("mmrelay.cli.get_config_paths")
+    @patch("os.path.isfile")
+    @patch("mmrelay.cli.validate_yaml_syntax")
+    @patch("mmrelay.cli._print_unified_e2ee_analysis")
+    @patch("mmrelay.e2ee_utils.get_e2ee_status")
+    @patch("builtins.print")
+    @patch("builtins.open", new_callable=mock_open)
+    def test_check_config_invalid_probe_timeout_non_finite(
+        self,
+        _mock_open,
```

Ruff (0.15.9)

[warning] Fixture `_mock_open` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)